src/components/pages/Search.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "../css/SearchPage.css";
+import Button from "../Button";
+
+function SearchBar(): JSX.Element {
```

Comment:
The elements look good but something seems off with the structure (i can't really point it out and can't think of any way to improve it). I am fine with merging this as it's a huge improvement over what was initially done, but maybe we could think of some ways to improve this a bit later after some discussion.

Reply:
I see. Visually, or code quality wise?

Reply:
Visually the elements look good, but the structure/how the elements are arranged could be improved. (I don't know how the laws of ui/ux design work so I'll leave that up to you)